### PR TITLE
Demo feedabacks and minor refactor

### PR DESCRIPTION
**Pull request by Kajish**
1. Added the functionality "Enter key press triggers submit" in OPM, OPM Comparison and Sessions page. 
2. Moved the Search bar from center to right end
3. Place holder for calendar in OPM page
4. Removed files "Order Status" , "Promotion"  from types and kept the type declarations in "Order Details" file

### CODE REVIEW

File: src/components/charts/BarChart.tsx [hunk 1]
```diff
@@ -196,6 +196,10 @@ const BarChart = () => {
     );
     setIsLoading(false);
   };
+  useEffect(() => {
+    const removeEventListener = submitOnEnter(incrementCounter);
+    return removeEventListener;
+  } );
```
Comment (raj-kumar-litmus) [inline src/components/charts/BarChart.tsx]: why does this useEffect run on every render ?

File: src/components/charts/BarChart.tsx [hunk 2]
```diff
@@ -196,6 +196,10 @@ const BarChart = () => {
     );
     setIsLoading(false);
   };
+  useEffect(() => {
+    const removeEventListener = submitOnEnter(incrementCounter);
```
Comment (raj-kumar-litmus) [inline src/components/charts/BarChart.tsx:201]: in `SubmitOnEnter`, you are passing a callback.
But `incrementCounter` is not a callback.
Please explain.

File: src/index.css [hunk 3]
```diff
@@ -613,6 +613,11 @@ hr.horizontalLine.blue {
 .calendarSessions span input {
   @apply !pl-[6.5vw] md:!pl-[2.5vw];
 }
+.calendarOpm span input {
+  @apply !pl-[6.5vw] md:!pl-[2.5vw];
+}
+
```
Comment (raj-kumar-litmus) [inline src/index.css]: Avoid un-necessary new lines.

File: src/components/charts/BarChart.tsx (hunk 2 above)
Comment (Kajish) [reply src/components/charts/BarChart.tsx:201]: im not getting entirely on what u meant over here  . const incrementCounter = () => {
    setSubmitCounter(submitCounter + 1);
  }; im passing this one into submitOnEnter.

File: src/index.css (hunk 3 above)
Comment (Kajish) [reply src/index.css]: Removed it

File: src/components/charts/BarChart.tsx (hunk 1 above)
Comment (Kajish) [reply src/components/charts/BarChart.tsx]: Added a dependency to it

**Review by thasnim-cm — APPROVED** [review]

**Review by sarath-sasi-panickaveetil — APPROVED** [review]
There are merge conflicts. Please check

**Review by sarath-sasi-panickaveetil — CHANGES_REQUESTED** [review]
Resolve conflicts

File: src/constants/appConstants.tsx [hunk 4]
```diff
@@ -127,30 +128,31 @@ export const DURATIONS = {
...
-  ALL: "",
-  DESKTOP: 0,
-  MOBILE_WEB: 5,
-  IPHONE_APP: 4,
-  ANDROID_APP: 9,
-  CSC: 2,
-  MPLUS: 6,
-  INSTAGRAM: 11,
+  All: "",
+  "Desktop": 0,
```
Comment (raj-kumar-litmus) [inline src/constants/appConstants.tsx]: remove white spaces at the end of lines.

File: src/components/orderDetails/OrderDetails.tsx [hunk 5]
```diff
@@ -127,7 +127,8 @@ const OrderDetails: React.FC = () => {
 
   const getPromotions = async () => {
     const promoUrl = URL_PROMOTIONS.replace(":orderId", orderId);
-    const data: OmsOrderFlow = await fetchData(promoUrl, {});
+    const data: IPromotion = await fetchData(promoUrl, {});
+    console.log(promotions);
```
Comment (sarath-sasi-panickaveetil) [inline src/components/orderDetails/OrderDetails.tsx]: remove console.log

**Pull request merged**